apis/js-apis/common-examples/record-attributions.md

```diff
@@ -52,6 +79,12 @@ nostojs(api => api
 .done()
 ```
 
-In the above example, we are recording a `View Product` event for product 7513863258337 which was clicked from the recommendation slot `productpage-nosto-3` while on another product page `7513872007393`.&#x20;
+In the above example,
+- `vp` specifies the type of event and it corresponds to View Product
+- `7513863258337` specifies the target and it corresponds to the ID of the product that's being viewed
+- `productpage-nosto-3` specifies the reference and it corresponds to the placement that hosted the product that's being clicked
+- `7513872007393` specifies the reference source and it corresponds to the ID of element, of the current page, that contained the `ref` element, `productpage-nosto-3` in this case
```

Comment:
```suggestion
- `7513872007393` specifies the reference source and it corresponds to the ID of element, of the current page, that contained the `ref` element, `productpage-nosto-3` in this case
```

`refSrc` for `vp` events refers to the id of the product that was shown where the recommendation was shown.  I think your original explanation explained it better.